wolfcrypt/src/signature.c

```diff
@@ -37,13 +39,43 @@
 
 /* If ECC and RSA are disabled then disable signature wrapper */
 #if !defined(HAVE_ECC) && defined(NO_RSA)
-#undef NO_SIG_WRAPPER
-#define NO_SIG_WRAPPER
+    #undef NO_SIG_WRAPPER
+    #define NO_SIG_WRAPPER
 #endif
 
 /* Signature wrapper disabled check */
 #ifndef NO_SIG_WRAPPER
 
+#if !defined(NO_RSA) && !defined(NO_ASN)
+static int wc_SignatureRsaEncode(enum wc_HashType hash_type, byte** hash_data,
```

Comment:
No sure this encode hash wrapper is unique to RSA.  That is, the Rsa naming may not be necessary.